app/assets/javascripts/dashboard.js

```diff
@@ -97,10 +122,32 @@ MinionPoller = {
     masterHtml = '<input name="roles[master][]" id="roles_master_' + minion.id +
       '" value="' + minion.id + '" type="radio" disabled="" ' + checked + '>';
 
+    statusText = ''
+
+    if (tx_update_reboot_nodes && minion.highstate == "applied") {
+      statusHtml = '<i class="fa fa-arrow-circle-up text-info fa-2x" aria-hidden="true"></i>';
+      statusText = 'Update Available'
+    }
+
+    if (tx_update_reboot_nodes && minion.highstate == "failed") {
```

Comment:
Do we want to take this as `Update Available` and silently hide the fact  that the highstate failed?